aiverify-foundation/moonshot
# [MS-331] Update Storage Return Path

## Description

Update the creation of object to return the path instead of a boolean so that we can update the users on where the object is created if required.

## Motivation and Context

QOL improvement

## Type of Change


A new feature

## How to Test

1. Start Moonshot in CLI.` python -m moonshot cli interactive`
2. [Positive case] Enter `export_bookmarks`. You should see the exported bookmarks path in your CLI
3. [Negative case] Rename the `bookmarks` folder to something else. Run the `export_bookmarks` command again. You should see a `[ERROR] ... Failed to export bookmarks` error printed out by the logger.

## Checklist

Please check all the boxes that apply to this pull request using "x":

- []  I have added or updated the necessary documentation (README, API docs, etc.).

## Screenshots (if applicable)

[If the changes involve visual modifications, include screenshots or GIFs that demonstrate the changes.]

## Additional Notes

[Add any additional information or context that might be relevant to reviewers.]

<details>
  <summary>Developer Certificate of Origin</summary>

 ```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
 ```
</details>
